Source/HYPScatterPlot.m

```diff
@@ -74,107 +80,109 @@ - (instancetype)initWithFrame:(CGRect)frame
                               @"upperThresholdYLabel": self.upperThresholdYLabel,
                               @"lowerThresholdYLabel": self.lowerThresholdYLabel,
                               @"plotView": self.plotView};
-
+    
     [self addConstraints:
      [NSLayoutConstraint constraintsWithVisualFormat:@"H:|-padding-[minimumXLabel]"
                                              options:0
                                              metrics:metrics
                                                views:views]];
-
+    
```

Comment:
Ewwww, have you tried https://github.com/zenangst/MarvinXcode? It trims whitespaces like this ones